assets/app/styles/_components.less

```diff
@@ -204,20 +204,23 @@
 }
 
 .pod-block {
-  .scaling-controls {
```

Comment:
My thought here might be to generalize `.scaling-controls` by taking it out of `.pod-block` and doing something like `<div class="scaling-controls md">` so we can reuse a sm/md/lg version in different places.  I don't think these styles need to be under `.pod-block`, this just happens to be first use case.